Penumbra/UI/SettingsInterface.cs

```diff
@@ -500,7 +501,96 @@ void DrawDeleteModal()
             }
 
             ImGui.EndPopup();
+        }
+
+        // Website button with On-Hover address if valid http(s), otherwise text.
+        private void DrawWebsiteText()
+        {
+            if ((_selectedMod.Mod.Meta.Website?.Length ?? 0) > 0)
+            {
+                var validUrl = Uri.TryCreate(_selectedMod.Mod.Meta.Website, UriKind.Absolute, out Uri uriResult)
+                    && (uriResult.Scheme == Uri.UriSchemeHttps ||uriResult.Scheme == Uri.UriSchemeHttp);
+                ImGui.SameLine();
+                if (validUrl)
+                {
+                    if (ImGui.SmallButton("Open Website"))
+                    { 
+                        Process.Start( _selectedMod.Mod.Meta.Website );
+                    }
+                    if (ImGui.IsItemHovered())
+                    {
+                        ImGui.BeginTooltip();
+                        ImGui.Text( _selectedMod.Mod.Meta.Website );
+                        ImGui.EndTooltip();
+                    }
+                }
+                else
+                {
+                    ImGui.TextColored( new Vector4( 1f, 1f, 1f, 0.66f ), "from" );
+                    ImGui.SameLine();
+                    ImGui.Text(_selectedMod.Mod.Meta.Website);
+                }
+            }
         }
+
+        // Create Mod-Handling buttons.
+        private void DrawEditButtons()
+        {
+            ImGui.SameLine();
+            if( ImGui.Button( "Open Mod Folder" ) )
+            {
+                Process.Start( _selectedMod.Mod.ModBasePath.FullName );
+            }
+
+            ImGui.SameLine();
+            if( ImGui.Button( "Edit JSON" ) )
+            {
+                var metaPath = Path.Combine( _selectedMod.Mod.ModBasePath.FullName, "meta.json");
+                File.WriteAllText( metaPath, JsonConvert.SerializeObject( _selectedMod.Mod.Meta, Formatting.Indented ) );
+                Process.Start( metaPath );
+            }
+
+            ImGui.SameLine();
+            if( ImGui.Button( "Reload JSON" ) )
+            {
+                ReloadMods();
+                _selectedMod = _plugin.ModManager.Mods.ModSettings[ _selectedModIndex ];

```

Comment:
this was probably how it was before, but this could potentially break if you delete a mod at the same time and hit this button